app/components/Button/Button.module.scss

```diff
@@ -0,0 +1,13 @@
+@import "../../styles/vars.scss";
+
+.button {
+  background: var(--button-background);
+  color: var(--button-foreground);
+  border-radius: $radius-full;
+  font-weight: 500;
+
```

Comment:
Thoughts on a transition here?

```suggestion
  transition: color,background-color .2s ease;

```

Reply:
+1 on the same transition as the icon button for dark mode toggle :)

Reply:
Ah should've known there would already be a transition in there 😏